### ISSUE TITLE
ZoneID error

### ISSUE THREAD

**Issue by superbunika** [MislavMandaric/home-assistant-vaillant-vsmart#29]
After updating to v0.2.0 through HACS I am now getting the following error in HA (v2021.11.3):

```
2021-12-02 21:43:20 ERROR (MainThread) [custom_components.vaillant_vsmart] Unexpected error fetching vaillant_vsmart data: 5 is not a valid ZoneId
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
self.data = await self._async_update_data()
File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 147, in _async_update_data
return await self.update_method()
File "/config/custom_components/vaillant_vsmart/entity.py", line 65, in _update_method
devices = await self._client.async_get_thermostats_data()
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 85, in async_get_thermostats_data
return [Device(**device) for device in body["body"]["devices"]]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 85, in <listcomp>
return [Device(**device) for device in body["body"]["devices"]]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 283, in __init__
self.modules = [Module(**module) for module in modules]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 283, in <listcomp>
self.modules = [Module(**module) for module in modules]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 322, in __init__
self.therm_program_list = [Program(**program) for program in therm_program_list]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 322, in <listcomp>
self.therm_program_list = [Program(**program) for program in therm_program_list]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 351, in __init__
self.zones = [Zone(**zone) for zone in zones]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 351, in <listcomp>
self.zones = [Zone(**zone) for zone in zones]
File "/usr/local/lib/python3.9/site-packages/vaillant_netatmo_api/thermostat.py", line 401, in __init__
self.id = ZoneId(id)
File "/usr/local/lib/python3.9/enum.py", line 384, in __call__
return cls.__new__(cls, value)
File "/usr/local/lib/python3.9/enum.py", line 702, in __new__
raise ve_exc
ValueError: 5 is not a valid ZoneId
```

**Comment by MislavMandaric** [comment]
Thanks for reporting this issue.

There is an issue when handling custom set profiles in the app. I just fixed the issue in the library and will push the new release to HACS soon.

**Comment by MislavMandaric** [comment]
Fixed in v0.2.1 release.
